Emby.Server.Implementations/Syncplay/SyncplayManager.cs

```diff
@@ -165,19 +175,28 @@ public void NewGroup(SessionInfo session)
 
             if (user.Policy.SyncplayAccess != SyncplayAccess.CreateAndJoinGroups)
             {
-                // TODO: shall an error message be sent back to the client?
-                throw new ArgumentException("User does not have permission to create groups");
+                _logger.LogWarning("Syncplaymanager NewGroup: {0} does not have permission to create groups.", session.Id);
+
+                var error = new GroupUpdate<string>()
+                {
+                    Type = GroupUpdateType.CreateGroupDenied
+                };
+                _sessionManager.SendSyncplayGroupUpdate(session.Id.ToString(), error, CancellationToken.None);
```

Comment:
maybe `NewGroup()` should also accept the cancellation token? I think that ideally the whole chain starting from the root HTTP request handler should pass down the cancellation token...

applicable here and everywhere else being called from public HTTP API